.github/workflows/testing.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+on: [pull_request]
+jobs:
+  unit_test_job:
+    name: Unit Tests
+    runs-on: macOS-latest
+    steps:
+      # Checks out a copy of your repository
+      - name: Checkout repository
+        uses: actions/checkout@v1
+      - name: Install gems
+        run: bundle install
```

Comment:
Can we do `bundle install cocoapods` since we're only using the `cocoapods` gem in the tasks below?

Reply:
Opted to use GH actions version of CocoaPods. Please comment if you prefer we use the version in our Gemfile.